x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -445,11 +467,26 @@ export class TelemetryEventsSender {
           ...(licenseId ? { 'X-Elastic-License-ID': licenseId } : {}),
         },
       });
+      this.telemetryUsageCounter?.incrementCounter({
+        counterName: createUsageCounterLabel(this.usageLabelPrefix.concat(['payloads', channel])),
```

Comment:
We may need to pass in additional metadata to this effect to collect the distribution of event types (diag, ep, etc)

Reply:
Ah ok, I can add that sort of case to the counter name itself.